src/uct/ib/rc/accel/rc_mlx5_common.c

```diff
@@ -206,6 +260,76 @@ unsigned uct_rc_mlx5_iface_srq_post_recv_ll(uct_rc_mlx5_iface_common_t *iface)
     return count;
 }
 
+unsigned uct_rc_mlx5_iface_srq_post_recv_sge(uct_rc_mlx5_iface_common_t *iface)
```

Comment:
Mostly same as `uct_rc_mlx5_iface_srq_post_recv`, may be make common and pass `set_seg` func to it?